src/docs/product/integrations/integration-platform/webhooks/errors.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Errors
+sidebar_order: 6
+description: "Learn more about Sentry's Integration Platform error webhooks."
+---
+
+Sentry integrations that have subscribed to error webhooks can receive information about an error being created.
+
+<Note>
+The error.created resource subscription is only available for Business plans and above.
```

Comment:
Wondering if this should be in code font?
```suggestion
The `error.created` resource subscription is only available for Business plans and above.
```

Reply:
It doesn't render properly :( I tried it in bold too and it just shows the ` and * as plaintext.

Reply:
> It doesn't render properly :( I tried it in bold too and it just shows the ` and * as plaintext.

Ah I know why. The lack of line breaks between the <Note> tags and the markdown.